genesis/engine/states/entities.py

```diff
@@ -43,36 +51,47 @@ def __init__(self, entity, s_global):
         self._s_global = s_global
 
         self._pos = gs.zeros(
```

Comment:
Define local variables for the "base shape" `(self.entity.sim._B, self._entity.n_particles)`, `requires_grad` and `scene`. And make all variable definitions one-line. This is taking to many lines, it is just unreadable.